Config.mak

```diff
@@ -9,3 +9,5 @@ override DFLAGS += -w -version=GLIBC
 ifdef CI
 COVFLAG := -cov
 endif
+
+INTEGRATIONTEST := test
```

Comment:
Why not moving the tests to `integrationtest/` to avoid the problems we wanted to avoid when we made the change in `makd`? :D

Reply:
Because it is ugly and ocean doesn't have that problem :(